### PR TITLE
Minor Performances Improvement:

### PR DESCRIPTION
**Pull request by arturobernalg**
* Replace 'StringBuffer' by 'StringBuilder'
* Duplicate throws
* Use Lambda
* Use single 'Map.computeIfAbsent' method call

### CODE REVIEW

File: core/misc/mal-elements-window/src/main/java/esa/mo/tools/mowindow/MOWindow.java [hunk 1]
```diff
@@ -130,12 +130,7 @@ private void interpretReceivedObj(Object obj, boolean editable) {
 
             }
 
-            java.awt.event.ActionListener actionListener = new java.awt.event.ActionListener() {
-                @Override
-                public void actionPerformed(java.awt.event.ActionEvent evt) {
-                    buttonAddActionPerformed(evt);
-                }
-            };
+            java.awt.event.ActionListener actionListener = evt -> buttonAddActionPerformed(evt);
```
[the remark this answers is not in the document]
Comment (arturobernalg) [reply core/misc/mal-elements-window/src/main/java/esa/mo/tools/mowindow/MOWindow.java:133]: Would you prefer that revert this change?

[the remark this answers is not in the document]
Comment (dmarszk) [reply core/misc/mal-elements-window/src/main/java/esa/mo/tools/mowindow/MOWindow.java:133]: Looks good.

**Comment by dmarszk** [comment]
Hi @arturobernalg - sorry for delayed processing of these. I started merging them but then noticed they are targeting master branch - can you rebase them no top of the dev ?

**Comment by dmarszk** [comment]
Hi @arturobernalg - sorry for delayed processing of these. I started merging them but then noticed they are targeting master branch - can you rebase them no top of the dev ?

**Comment by arturobernalg** [comment]
> Hi @arturobernalg - sorry for delayed processing of these. I started merging them but then noticed they are targeting master branch - can you rebase them no top of the dev ?

Done
TY

**Pull request merged**